src/main/java/com/faforever/api/data/domain/Avatar.java

```diff
@@ -22,6 +34,16 @@
 @Entity
 @Table(name = "avatars_list")
 @Include(rootLevel = true, type = Avatar.TYPE_NAME)
+@Indexed
+@AnalyzerDef(name = "case_insensitive",
```

Comment:
It would be nice if this could be defined elsewhere as this is global definition not bound to entity. I've searched for solution but did not find one yet